specification/standbypool/StandbyPool.Management/service.tsp

```diff
@@ -43,3 +49,13 @@ union ProvisioningState {
   @doc("Resource is being deleted.")
   Deleting: "Deleting",
 }
+
+@added(Microsoft.StandbyPool.Versions.`2024-03-01-preview`)
+@doc("Displays the counts of pooled resources in each state, as known by the StandbyPool resource provider.")
+model PoolResourceStateCount {
+  @doc("The state that the pooled resources count is for.")
+  state: string;
```

Comment:
>string [](http://example.com/codeflow?start=9&length=6)

couldnt this property an enum ?

Reply:
My coworker owned this feature, he set this as string because he this we will introduce other state in the future. 

Do you think we should make this as an enum or keep using string is fine? Open for suggestions.

![image](https://github.com/user-attachments/assets/28637a57-eab0-4422-9104-e35216b4633b)

Reply:
This was from a time when we were not using modelAsString:true for enums. We discovered we can use that to add values to enums without it being a breaking change, as is done here: https://github.com/Azure/azure-rest-api-specs-pr/pull/18956

This could be an enum, but is changing string -> enum a breaking change?

Reply:
as this pr is for moving an existing version you can keep it like this and make the update in the next preview version